base-libs/expat/autobuild/defines

```diff
@@ -2,3 +2,5 @@ PKGNAME=expat
 PKGDEP="glibc"
 PKGSEC=libs
 PKGDES="A stream-oriented C library for parsing XML"
+
+AUTOTOOLS_AFTER=" --disable-static"
```

Comment:
Just leave this out, there is no reason to take out static library. Over time, we would even want to make it a default to ship static libraries - we haven't had a consensus on this yet.